src/05-objects.js

```diff
@@ -49,7 +59,11 @@ function createObjectWithKey(key) {}
  *  createObjectWithKeyValuePair("left", "right");
  *  //> { left: "right" }
  */
-function createObjectWithKeyValuePair(key, val) {}
+function createObjectWithKeyValuePair(key, val) {
+  return {
+    [key]: val
```

Comment:
Oooohhh nice! We haven't shown you all this but its definitely a fix for this issue and I'm glad you found/used it! 🔥